x-pack/plugins/monitoring/public/alerts/lib/alerts_toast.tsx

```diff
@@ -22,13 +22,13 @@ const showApiKeyAndEncryptionError = () => {
   const settingsUrl = Legacy.shims.docLinks.links.alerting.generalSettings;
 
   Legacy.shims.toastNotifications.addWarning({
-    title: toMountPoint(
+    title: mountReactNode(
       <FormattedMessage
         id="xpack.monitoring.healthCheck.tlsAndEncryptionErrorTitle"
         defaultMessage="Additional setup required"
       />
     ),
```

Comment:
This is one of the places where we can use `i18n.translate` instead of the `<FormattedMessage` component.